src/app/(views)/github/t/@pullButtons/pulls/page.tsx

```diff
@@ -14,12 +17,21 @@ import {
   useUpdateRepoOrPull
 } from '@/contexts/github/root/selected-pulls-provider'
 
+export enum ESettingsCode {
+  EXPAND_ALL_BUTTON = 'EXPAND_ALL_BUTTON',
+  SELECT_ALL_DEPENDABOT_PULL_REQUESTS = 'SELECT_ALL_DEPENDABOT_PULL_REQUESTS',
+  START_REVIEW_BUTTON = 'START_REVIEW_BUTTON'
+}
+
+const DEPENDABOT_USER_NAME = 'dependabot[bot]'
+
 export default function Page() {
   const { repos } = useRepos()
   const flattenCheckedPulls = getFlattenCheckedPulls(repos)
   const { translate } = useDictionary()
   const [isDialogOpen, setIsDialogOpen] = useState<boolean>(false)
-  const { openAllRepo } = useUpdateRepoOrPull()
+  const [hasDependabotPulls, setHasDependabotPulls] = useState<boolean>(false)
```

Comment:
No need `const [hasDependabotPulls, setHasDependabotPulls] = useState<boolean>(false)`. If you are having more state that is not needed, it's going to be difficult to debug in the future